suite-native/storage/src/createAsyncMigrate.ts

```diff
@@ -0,0 +1,70 @@
+import { DEFAULT_VERSION, PersistedState } from 'redux-persist';
+
+export type UnknownPersistedState = unknown & PersistedState;
+
+export type MigrationsManifest = {
+    [key: string]: (
```

Comment:
What about typing the key as number? I believe it would be safer since we are counting on the numerical order. Now it would be possible to set migrations with the string keys and that could bring some troubles. But probably very nitpicky comment. I can not image why would someone try to assign string key to migrations object.
```suggestion
    [key: number]: (
```

Reply:
Agree, it would be better for readability to make it consistent and allow only numbers 👌

Reply:
bfc2409d7392725f93e46fd844ba49a5c6d3c347